c/le.c

```diff
@@ -121,7 +121,12 @@ void abortIfUnsupportedCAA() {
 #ifndef METTLE
   if (zosVersion > LE_MAX_SUPPORTED_ZOS) {
     const char *continueWithWarning = getenv("ZWE_zowe_launcher_unsafeDisableZosVersionCheck");
-    if (!strcmp(continueWithWarning, "true")) {
+    /* using STDENV in JCL can result in env values that need trimming... */
+    char *trimmed[strlen(continueWithWarning)+1];
+    strncpy(trimmed, continueWithWarning, strlen(continueWithWarning));
```

Comment:
This will leave the destination unterminated because `trimmed isn't initialized to zeroes; `trimRight` may 0C4.